plugins/ports-plugin/src/port-scanner.ts

```diff
@@ -29,8 +29,12 @@ export class PortScanner {
         const ipConverter = new IpConverter();
         // connect to /proc/net/tcp and /proc/net/tcp6
         const command = new Command(__dirname);
-        const outputv4 = await command.exec(PortScanner.GRAB_PORTS_IPV4);
-        const outputv6 = await command.exec(PortScanner.GRAB_PORTS_IPV6);
+        const outputv4 = await command
```

Comment:
hello, wouldn't it be better to check if file is there instead of running the command and failing and display the error as it will be displayed tons of time

Reply:
Hmm. An another side question. What is the reason why `fs` package isn't used here?  There is a room for consideration to stop using `command` if my current patch should be more integrated.

Reply:
Code was totally refactored. So I mark this tread as resolved.